packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js

```diff
@@ -0,0 +1,108 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { View } from 'react-native';
+
+import ActionSheet from 'react-native-actionsheet';
+import {
+  Avatar,
+  BodyText,
+  Card,
+  CardContent,
+  H3,
+  H6,
+  PaddedView,
+  styled,
+  Touchable,
+  ButtonLink,
+  ChannelLabel,
+} from '@apollosproject/ui-kit';
+
+const StyledCard = styled(() => ({
+  marginHorizontal: 0,
+  marginVertical: 0,
+}))(Card);
+
+const EllipsisView = styled(({ theme }) => ({
+  paddingHorizontal: theme.sizing.baseUnit,
+}))(View);
+
+const GreyH3 = styled(({ theme }) => ({
+  color: theme.colors.text.tertiary,
+  textAlign: 'right',
+}))(H3);
+
+const GreyH6 = styled(({ theme }) => ({
+  color: theme.colors.text.tertiary,
+}))(H6);
+
+const StyledCardContent = styled(() => ({
+  alignItems: 'center',
+}))(CardContent);
+
+const StyledBodyText = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 1.5,
```

Comment:
Per all the other reviews, we should probably avoid putting margin (or other spacing related styles) on actual typography components. Instead, put the margin on wrapping components.